service/src/main/java/io/camunda/service/license/CamundaLicense.java

```diff
@@ -37,6 +46,14 @@ public synchronized LicenseType getLicenseType() {
     return licenseType;
   }
 
+  public synchronized boolean isCommercial() {
+    return isCommercial;
+  }
+
+  public synchronized String expiresAt() {
+    return DATE_TIME_FORMATTER.format(validUntil);
+  }
+
```

Comment:
🔧 Let the consumers format this information as needed. The license shouldn't care about this.